zenoh/__init__.pyi

```diff
@@ -101,18 +101,54 @@ class Encoding:
     """Just some bytes.
     
     Constant alias for string: "zenoh/bytes"."""
-    ZENOH_INT: Self
-    """A VLE-encoded signed little-endian integer. Either 8bit, 16bit, 32bit, or 64bit. Binary reprensentation uses two's complement.
+    ZENOH_INT8: Self
```

Comment:
Just to double check. Docstrings go above or below the constants? In this case all the docstrings are below.

Reply:
From [PEP 257](https://peps.python.org/pep-0257/)
> String literals occurring immediately after a simple assignment at the top level of a module, class, or __init__ method are called “attribute docstrings”.